app/backend/approaches/chatreadretrieveread.py

```diff
@@ -273,16 +271,24 @@ async def run_until_final_call(
                 ),
             ],
         }
+
+        # If the temperature is overridden via the API request, use that value.
+        # Otherwise, use the default value from the model configuration.
+        # If model configuration does not have a temperature, use the default value of 0.3.
+        if overrides.get("temperature") is not None:
+            chat_prompty._model.parameters["temperature"] = overrides.get("temperature")
+        else:
+            chat_prompty._model.parameters.setdefault("temperature", 0.3)
```

Comment:
Is the temperature the only value that we allow overriding dynamically?

Reply:
More or less yes, however you are right, there should be a dynamic way to support the overrides.

Reply:
Comment for myself: 
done